src/core/Global.ttslua

```diff
@@ -418,7 +424,8 @@ function returnAndRedraw(_, tokenGUID)
 
   -- perform the actual token replacing
   trackChaosToken(tokenName, mat.getGUID(), true)
-  returnChaosTokenToBag(returnedToken)
+  params = {token = returnedToken, fromBag = true}
```

Comment:
Shouldn't be global. Could also just create this inline:
```lua
returnChaosTokenToBag({ token = returnedToken, fromBag = true })
```

Reply:
made them local to avoid changing other things too